pkgs/applications/misc/inherd-quake/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+, stdenv
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "quake";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "phodal";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256:1f7k68g18g3dpnrsmhgmz753bly1i3f4lmsljiyp9ap0c6w8ahgg";
+  };
+
+  cargoSha256 = "sha256:1yqj9rq770j116138bqn4ycggy13vvym1cz50myfddb9rjjzafrl";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl ] ++ (
+    lib.optional stdenv.isDarwin [
+      darwin.apple_sdk.frameworks.Security
+      darwin.apple_sdk.frameworks.CoreServices
+    ]
+  );
```

Maybe use <code>lib.optional<b>s</b></code>?

```suggestion
  buildInputs = [
    openssl
  ] ++ lib.optionals stdenv.isDarwin [
    CoreServices
    Security
  ];
```